src/types.ts

```diff
@@ -0,0 +1,195 @@
+export enum Curve {
+    ed25519 = 'ed25519',
+    P256 = 'P256',
+    secp256k1 = 'secp256k1',
+}
+
+export enum HashAlgorithm {
+    sha256 = 'sha256',
+    sha512 = 'sha512',
+    sha3_256 = 'sha3-256',
+    sha3_512 = 'sha3-512',
+    blake2b = 'blake2b',
+}
+
+export enum UserTypeTransaction {
+    keychain = 'keychain',
+    keychain_access = 'keychain_access',
+    transfer = 'transfer',
+    hosting = 'hosting',
+    token = 'token',
+    data = 'data',
+    contract = 'contract',
+    code_proposal = 'code_proposal',
+    code_approval = 'code_approval',
+}
+
+enum NetworkTypeTransaction {
```

Comment:
Why do you need this kind of enum ?

Reply:
libjs doesn't require it but developers might need it